src/components/ListItem.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import Checkbox from '../components/Checkbox';
 
-const ListItem = ({ item, shoppingList }) => (
-  <li className="list__item">
+const ListItem = ({ index, item, shoppingList }) => (
+  <li className={index} aria-label={index}>
```

Comment:
Hi @C-Zekeri. So far, all aria-labels are ‘inactive’ but according to the AC, items in different states should be described distinctly?

Reply:
They are. You probably need to fiddle with the db a bit, and ensure that items have been purchased at least twice to see the other states. Same goes for colour.

Reply:
About the legend, would be a nice touch I agree. I'll ask the mentors if it's necessary, thank you.

Reply:
Added it, thanks.